docs/source/deploying.rst

```diff
@@ -93,62 +92,83 @@ Those workers are then allocated physical hardware resources.
    client = Client(cluster)  # Connect to distributed cluster and override default
    df.x.sum().compute()  # This now runs on the distributed system
 
-.. figure:: images/dask-cluster-manager.svg
-   :scale: 50%
-
-   An overview of cluster management with Dask distributed.
-
 .. _deployment-options:
 
-To summarize, you can use the default, single-machine scheduler to use Dask
-on your local machine. If you'd like use a cluster *or* simply take advantage
-of the :doc:`extensive diagnostics <../diagnostics-distributed>`,
-you can use Dask distributed. The following resources explain
-in more detail how to set up Dask on a variety of local and distributed hardware:
+The following resources explain how to set up Dask on a variety of local and distributed hardware.
 
```

Comment:
@gjoseph92 I made a first pass at your suggestions in https://github.com/dask/dask/pull/9912#pullrequestreview-1284106497 but not sure I have the categories quite right.

I also tried a couple tab variations, I think the list seemed a bit long with tabs for individual/team/IT, but maybe we can condense it?
![Screen Shot 2023-02-15 at 6 47 34 PM](https://user-images.githubusercontent.com/8620816/219256049-191233ea-9239-41a1-925a-da07b8efab8b.png)

Reply:
I also tried nested tabs, but those seem a bit busy:
![Screen Shot 2023-02-15 at 6 11 52 PM](https://user-images.githubusercontent.com/8620816/219256166-23d9c8b8-79e6-46c8-becc-0bc0168a21dc.png)

Reply:
I think it would only be individuals / teams; not sure what things would go under "For IT".

I do still like the idea of this, but I agree that I'm not sure how to organize it. Probably the first option (just one set of tabs)?

Reply:
Thanks @gjoseph92! I left it as-is (without tabs) for now, but happy to revisit this.